frontend/public/locales/en/common.json

```diff
@@ -202,5 +202,6 @@
   "vending-onboard": "Create a Stripe Express account",
   "requires-attention": "Requires attention",
   "cancel": "Cancel",
-  "delete": "Delete"
+  "delete": "Delete",
+  "go-to-page": "Go to {{page}}"
```

Comment:
I'm torn on this, I kinda think doing this in a breadcrumb style might be easier to handle, but not super sure I want to push that onto you.

Reply:
From a UX perspective that sounds like the better solution, unsure if it fits with our work packages but perhaps something I'd look at in my free time if nobody else is inclined. Say we push it out to a new issue for now?